.changeset/proud-cycles-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/backend': patch
+---
+
+Avoid 401 responses for satellite apps for non-browser requests
```

Comment:
Feel free to ignore :) 
I think the comment here is somewhat misleading, as the user might still decide to return a 401 "unauthorized" response themselves. 

Something along the lines of 
`Treat expired JWT as signed-out state for requests originated from non-browser clients on satellite apps`
might make more sense?